generators/advanced/templates/root/_Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:10.4
```

Comment:
I will suggest to keep nvm and Docker aligned to v12. 

I will suggest to use docker multistage to [avoid vulnerabilities and overweight](https://snyk.io/blog/top-ten-docker-images-contain-over-8000-vulnerable-paths/)

Reply:
right new [issue](https://github.com/guidesmiths/generator-systemic/issues/53)